examples/python/policies/create_account_policy.py

```diff
@@ -0,0 +1,38 @@
+# Usage: uv run python policies/create_account_policy.py
+
+import asyncio
+
+from cdp import CdpClient
+
```

Comment:
examples need to use dotenv now
```suggestion
from dotenv import load_dotenv

load_dotenv()
```